src/modules/datacenter.js

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios'
+
+const getMarketCapData = async () => {
+  let today = new Date()
+  today.setDate(today.getDate() - 7)
+  today = today.toISOString().split('today')[0]
+
+  let response = await axios.get(`/api/markets?date=${today}`);
+  return response.data
+
+
+
```

Comment:
Please remove extra lines and use `camelCase` in file name. `datacenter` should be `dataCenter`